### PR TITLE
Switch base images back to dockerhub

### PR DESCRIPTION
**Pull request by ialidzhikov**
/kind enhancement

Follow-up PR after https://github.com/gardener/gardener-extension-os-coreos/pull/19.

**Release note**:

```improvement operator
NONE
```

### CODE REVIEW

**Comment by ialidzhikov** [comment]
I see that we need to adapt the PR status checks - `concourse-ci/publish` is obsolete now and replaced by `concourse-ci/build_oci_image_gardener-extension-os-coreos`. Can someone from this repo's administrators help with it? 
The related github docs - ref https://docs.github.com/en/github/administering-a-repository/managing-a-branch-protection-rule (point 7).

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-FROM eu.gcr.io/gardener-project/3rd/golang:1.15.7 AS builder
+FROM golang:1.15.8 AS builder
 
 WORKDIR /go/src/github.com/gardener/gardener-extension-os-coreos
 COPY . .
 RUN make install
 
 ############# gardener-extension-os-coreos
-FROM eu.gcr.io/gardener-project/3rd/alpine:3.12.3 AS gardener-extension-os-coreos
+FROM alpine:3.12.3 AS gardener-extension-os-coreos
```
Comment (vpnachev) [inline Dockerfile:9]:
```suggestion
FROM alpine:3.13.2 AS gardener-extension-os-coreos
```

**Review by vpnachev — APPROVED** [review]
/lgtm

While you are touching the alpine image, could you also update it to the latest version :wink:

**Review by vpnachev — APPROVED** [review]
/lgtm

**Pull request merged**